# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/ron/Cargo.toml
# Commit: 3f167f8b3137
# Full Hash: 3f167f8b313749f50f135e189e5854bef7ac0c73
# Author: Erich Gubler <erichdongubler@gmail.com>
# Date: 2025-10-03 18:08 +0000
# Regressor Bug: 1991551
# File Overlap Count: 23
# Overlapping Files: third_party/rust/wgpu-hal/Cargo.toml, supply-chain/audits.toml, third_party/rust/naga/src/back/spv/writer.rs, third_party/rust/naga/src/back/msl/writer.rs, third_party/rust/wgpu-types/src/lib.rs
# Description:
#   Bug 1991551 - build(webgpu): update WGPU to 8c4aebc0c1b75e3f7412dfc180745a0717fc30ec r=webgpu-reviewers,supply-chain-reviewers,nical
# ==============================================================================

diff -r 823439f307a0 -r 3f167f8b3137 third_party/rust/ron/Cargo.toml
--- a/third_party/rust/ron/Cargo.toml	Fri Oct 03 18:01:30 2025 +0000
+++ b/third_party/rust/ron/Cargo.toml	Fri Oct 03 18:08:29 2025 +0000
@@ -13,7 +13,7 @@
 edition = "2021"
 rust-version = "1.64.0"
 name = "ron"
-version = "0.10.1"
+version = "0.11.0"
 authors = [
     "Christopher Durham <cad97@cad97.com>",
     "Dzmitry Malyshau <kvarkus@gmail.com>",
@@ -47,8 +47,14 @@
 rustdoc-args = ["--generate-link-to-definition"]
 
 [features]
-default = []
+default = ["std"]
+indexmap = [
+    "std",
+    "dep:indexmap",
+]
 integer128 = []
+internal-span-substring-test = ["unicode-segmentation"]
+std = ["serde/std"]
 
 [lib]
 name = "ron"
@@ -57,26 +63,36 @@
 [[example]]
 name = "base64"
 path = "examples/base64.rs"
+required-features = []
 
 [[example]]
 name = "decode"
 path = "examples/decode.rs"
+required-features = []
 
 [[example]]
 name = "decode_file"
 path = "examples/decode_file.rs"
+required-features = ["std"]
 
 [[example]]
 name = "encode"
 path = "examples/encode.rs"
+required-features = []
 
 [[example]]
 name = "encode_file"
 path = "examples/encode_file.rs"
+required-features = ["std"]
+
+[[example]]
+name = "file_read_write_vec"
+path = "examples/file_read_write_vec.rs"
 
 [[example]]
 name = "transcode"
 path = "examples/transcode.rs"
+required-features = []
 
 [[test]]
 name = "115_minimal_flattening"
@@ -279,6 +295,10 @@
 path = "tests/544_path_meta.rs"
 
 [[test]]
+name = "564_exhaustive_number"
+path = "tests/564_exhaustive_number.rs"
+
+[[test]]
 name = "big_struct"
 path = "tests/big_struct.rs"
 
@@ -356,7 +376,7 @@
 
 [dependencies.base64]
 version = "0.22"
-features = ["std"]
+features = ["alloc"]
 default-features = false
 
 [dependencies.bitflags]
@@ -366,16 +386,13 @@
 
 [dependencies.indexmap]
 version = "2.0"
-features = [
-    "std",
-    "serde",
-]
+features = ["serde"]
 optional = true
 default-features = false
 
 [dependencies.serde]
 version = "1.0.181"
-features = ["std"]
+features = ["alloc"]
 default-features = false
 
 [dependencies.serde_derive]
@@ -386,6 +403,11 @@
 version = "1.0"
 default-features = false
 
+[dependencies.unicode-segmentation]
+version = "1.12.0"
+optional = true
+default-features = false
+
 [dev-dependencies.bytes]
 version = "1.3"
 features = ["serde"]